templates/zerver/help/view-someones-profile.md

```diff
@@ -10,21 +10,16 @@ streams for which [you can see all subscribers](/help/stream-permissions).
 
 ## View someone's profile
 
-### Via the message list
-
 {start_tabs}
 
-1. Click on a user's profile picture.
-
-1. Click **View full profile**.
-
-{end_tabs}
+{tab|desktop-web}
 
-### Via the right sidebar
+{!right-sidebar-view-full-profile.md!}
 
-{start_tabs}
+!!! Tip ""
 
-{!right-sidebar-view-full-profile.md!}
+    You can also click on a user's profile picture or name and then
+    click **View full profile**.
```

Comment:
I wonder if we could put this tip above the "View full profile" instruction to be able to drop the "View full profile" part of it? Can we try that and see if it looks confusing?

Reply:
That's a good idea, but unfortunately, it would break up the numbering i.e. the number in step three would get rendered as a number `1` instead of `3`.